libraries/entities-renderer/src/RenderablePolyVoxEntityItem.cpp

```diff
@@ -238,53 +238,6 @@ void RenderablePolyVoxEntityItem::setVoxelSurfaceStyle(PolyVoxSurfaceStyle voxel
     startUpdates();
 }
 
-glm::vec3 RenderablePolyVoxEntityItem::getSurfacePositionAdjustment() const {
```

Comment:
these methods are duplicates of methods that were already in PolyvoxEntityItem.cpp, so they can simply be removed